collector/prometheus/src/main/java/org/apache/karaf/decanter/collector/prometheus/PrometheusCollector.java

```diff
@@ -71,28 +73,39 @@ public void activate(Dictionary<String, Object> properties) throws Exception {
     public void run() {
         try {
             URLConnection connection = prometheusURL.openConnection();
-            Map<String, Object> data = new HashMap<>();
-            data.put("type", "prometheus");
+            String topic = (properties.get(EventConstants.EVENT_TOPIC) != null) ? (String) properties.get(EventConstants.EVENT_TOPIC) : "decanter/collect/prometheus";
+            String type = (properties.get("type") != null) ? (String) properties.get("type") : "prometheus";
```

Comment:
Same, already possible.

That said, I think it's important to have consistency in `type`, so why not having hardcoded `type` here.